servicetalk-grpc-api/src/main/java/io/servicetalk/grpc/api/GrpcRouter.java

```diff
@@ -100,12 +98,11 @@
  * implementation of a <a href="https://www.grpc.io">gRPC</a> method.
  */
 final class GrpcRouter {
-    private static final Logger LOGGER = LoggerFactory.getLogger(GrpcRouter.class);
```

Comment:
At some point, this logger produced a nice debugging output:
```
2021-11-02 15:17:37,124                           main [DEBUG] GrpcRouter                     - route strategy for path=/helloworld.Greeter/SayHello : ctx=DEFAULT_HTTP_EXECUTION_STRATEGY route=null use=DEFAULT_HTTP_EXECUTION_STRATEGY
```
Don't see it anymore. Consider adding it back. 

For `use=` consider either logging a result of the merge or just say something like: using a combination of both.